src/config.rs

```diff
@@ -228,15 +229,28 @@ impl Config {
         }
 
         let daemon_dir = &config.daemon_dir;
-        let daemon_cookie_file = config
-            .cookie_file
-            .unwrap_or_else(|| daemon_dir.join(".cookie"));
+        let daemon_auth = match (config.auth, config.cookie_file) {
+            (None, None) => Auth::CookieFile(daemon_dir.join(".cookie")),
+            (None, Some(cookie_file)) => Auth::CookieFile(cookie_file),
+            (Some(cookie), None) => {
```

Comment:
Nit: I'd prefer calling this `auth` instead of `cookie` - same reason we renamed the parameter. :)

Reply:
Good catch, thanks!

Reply:
Fixed in https://github.com/romanz/electrs/pull/442/commits/8ef15e25b9d24e16626a6157b07de9d716162dce